src/include/internal_libreport.h

```diff
@@ -729,7 +729,19 @@ int get_env_variable(pid_t pid, const char *name, char **value);
 #define PROC_NS_ID_USER 4
 #define PROC_NS_ID_UTS 5
 #define PROC_NS_ID_CGROUP 6
-static const char * libreport_proc_namespaces[] = { "ipc", "mnt", "net", "pid", "uts", "user", "cgroup", "pid_for_children" };
+#define PROC_NS_ID_TIME 8
+static const char * libreport_proc_namespaces[] = {
+    "ipc",
+    "mnt",
+    "net",
+    "pid",
+    "uts",
+    "user",
+    "cgroup",
+    "pid_for_children",
+    "time",
+    "time_for_children",
```

Comment:
I might be overly pedantic – would it make sense to have them sorted in alphabetic order?

Reply:
Well, my concern here is binary compatibility, what with the internal header not being internal. And it does seem that ABRT uses some of the indices defined above, so we would need to rebuild it.

Reply:
Actually, since we’re already breaking it with Michal’s changes, I might as well reorder these.